src/main/java/PalindromeChecker.java

```diff
@@ -3,11 +3,14 @@
 
     public boolean check(String word) {
 
+        //still needed?
         if(word.length() == 1) {
             return true;
         }
 
+        //reassigning? - just question-marks?
```

Comment:
I'd avoid reassigning a value to the parameter, what if you need to use the original value?  Plus it can be confusing and lead to bugs.  Much better to just declare a new local variable to hold the modified version.